commands-yml/commands/session/other/update-settings.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Get Device Settings
+short_description: Update the current setting on the device
+example_usage:
+  javascript_wd:
+    |
+      await driver.settings();
+  javascript_wdio:
+    |
+      let settings = driver.settings();
+
+client_docs:
+  javascript_wdio: "http://webdriver.io/api/mobile/settings.html"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3018"
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: true
+    uiautomation: true
+  android:
+    uiautomator2: true
+    uiautomator: true
+  mac:
+    mac: true
+  windows:
+    windows: true
+client_support:
```

Comment:
If all these are true, why no examples?

Reply:
@vmaxim @SrinivasanTarget do either of you know if this command is supported in Java client?

Reply:
yes it is supported @dpgraham